docs/developer_notes/contributing.rst

```diff
@@ -42,6 +77,26 @@ For external contributors
 
 6. Open a pull request from your fork to our main branch
 
+7. Before you push, there might be new changes in the repo to
+   integrate. To merge the changes::
+   
+    git fetch origin
+    git checkout feature-name  --does nothing if it's your branch already
```

Comment:
Consider replacing `--` with `#` so that the command is still valid

Reply:
done